src/lib.rs

```diff
@@ -776,6 +804,48 @@ impl<T> Slab<T> {
         }
     }
 
+    /// Returns mutable references to many indices at once.
+    /// Returns [`GetDisjointMutError`] if the indices are out of bounds,
+    /// overlapping, or vacant.
```

Comment:
Summary should be rendered on its own line in docs.
```suggestion
    /// Returns mutable references to many indices at once.
    ///
    /// Returns [`GetDisjointMutError`] if the indices are out of bounds,
    /// overlapping, or vacant.
```